index.js

```diff
@@ -891,6 +923,12 @@
     };
   }
 
+  //  Function$prototype$compose:: (b -> c) ~> (a -> b) -> (a -> c)
```

Comment:
I'm not sure about this signature. Did I get it wrong in https://github.com/fantasyland/fantasy-land/pull/248?